org.kde.kontact.json

```diff
@@ -2,18 +2,18 @@
     "id": "org.kde.kontact",
     "rename-icon": "kontact",
     "runtime": "org.kde.Platform",
-    "runtime-version": "6.6",
+    "runtime-version": "6.7",
     "base": "io.qt.qtwebengine.BaseApp",
-    "base-version": "6.6",
+    "base-version": "6.7",
     "sdk": "org.kde.Sdk",
     "sdk-extensions": [
         "org.freedesktop.Sdk.Extension.rust-stable"
     ],
-    "command": "run_kontact.sh",
+    "command": "kontact",
```

Comment:
I do not like this, the old way makes it super clear that a script that is doing more things than run kontact is happening, this is hidden with a rename very down in the file, what's the benefit of this?